### PR TITLE
Add "linux" selector to sdn-ovs.yaml

### PR DESCRIPTION
**Pull request by pecameron**
Add beta.kubernetes.io/os: "linux"
(This was ommitted in PR 425)

Signed-off-by: Phil Cameron <pcameron@redhat.com>

### CODE REVIEW

**Comment by pecameron** [comment]
@rajatchopra @dcbw @squeed PTAL

File: dist/yaml/sdn-ovs.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ spec:
         component: network
         type: infra
         openshift.io/component: network
+        beta.kubernetes.io/os: "linux"
```
Comment (rajatchopra) [inline dist/yaml/sdn-ovs.yaml]: This label is not necessary.
/lgtm

**Comment by dcbw** [comment]
@pecameron can you respin to remove the label rajat talks about?

**Comment by squeed** [comment]
Hang on - from my perspective, that label *should* be there, no?

**Comment by pecameron** [comment]
@rajatchopra @dcbw @squeed the label is on the other daemonsets. Why not this one?

**Comment by squeed** [comment]
Wait, nevermind. I don't think the label is necessary, just the NodeSelector.

**Comment by rajatchopra** [comment]
The label isn't needed on other daemonsets either. We just need the node selector.
In offline emails, I am guilty at least once of writing 'labels' when I really meant 'node selector'. Sorry about the confusion.

**Comment by pecameron** [comment]
@dcbw @rajatchopra @squeed  made the changes, PTAL

**Pull request merged**